src/main/java/io/pivotal/cfapp/domain/AppDetailShim.java

```diff
@@ -30,6 +30,10 @@
 	private String lastEventActor;
 	private LocalDateTime lastEventTime;
 	private String requestedState;
+	private String buildpackReleaseType;
+	private LocalDateTime buildpackReleaseDate;
```

Comment:
I wonder if this should really be a `LocalDate`. No change required.  Just a question.

Reply:
LocalDateTime not required; LocalDate will suffice the ask. For consistency, I used LocalDateTime.